content/articles/why-we-dont-do-webhosting.markdown

```diff
@@ -0,0 +1,18 @@
+---
+title: Why we don't offer web hosting services.
+excerpt: Welcome to DNSimple. This support page explains why we don't offer web hosting along our DNS hosting.
+categories:
+- DNSimple
+---
+
+# Why we don't offer web hosting services
+
+Traditionally DNS providers offered a basic web hosting solution along the DNS hosting service. It usually translated into giving several Megabytes of disk space served by an HTTP server and accessible with a FTP client to the user for HTML static sites. 

```

Comment:
Would it be better to turn this sentence around and state that web hosting and DNS are usually bundled together? Or maybe point out the link that often exists between registrars and DNS?